src/Sylius/Bundle/CoreBundle/spec/Checkout/CheckoutProcessScenarioSpec.php

```diff
@@ -51,7 +52,10 @@ function it_builds_checkout_process_with_proper_steps(ProcessBuilderInterface $b
         $builder->setDisplayRoute('sylius_checkout_display')->willReturn($builder)->shouldBeCalled();
         $builder->setForwardRoute('sylius_checkout_forward')->willReturn($builder)->shouldBeCalled();
 
-        $builder->setRedirect('sylius_homepage')->willReturn($builder)->shouldBeCalled();
+        $builder->setRedirect('sylius_account_order_show')->willReturn($builder)->shouldBeCalled();
+        $builder->setRedirectParams(array(
+            'number' => $order->getNumber()
+        ))->willReturn($builder)->shouldBeCalled();
```

Comment:
Actually we do want to check whether the method was called with this arguments or not, so the `shouldBeCalled()` is necessary here, because `willReturn` doesn't validate that method was called.